package.json

```diff
@@ -22,6 +22,7 @@
         "lint": "dtslint types"
     },
     "devDependencies": {
+        "@types/node": "^12.12.14",
```

Comment:
Why was this needed?

Reply:
It wasn't sorry! I updated the pull request, let me know if I have to update anything!

Reply:
Thanks!